rollup.config.js

```diff
@@ -116,6 +118,13 @@ const baseConfig = {
       extract: 'components.css',
       autoModules: false,
       modules: {generateScopedName: 'prc_[local]-[hash:base64:5]'},
+      plugins: [
+        postcssCustomPropertiesFallback({
+          importFrom: {
+            customProperties: importedJSONFromPrimitives,
+          },
+        }),
+      ],
```

Comment:
Are these fallbacks for a "default" theme? Would it be possible to have these fallbacks be dynamic for whatever the product specifies as the default theme? e.g. in Sass this might look like:

```scss
// Exporting tokens in package
@use 'theme';

$bgColor-default: var(--bgColor-default, theme.get('bgColor-default'));

// Product usage
@use '@primer/styles/scss/themes';
@use '@primer/styles/scss/theme' with (
  $theme: themes.$dark,
);
@use '@primer/styles/scss/tokens' as *;

.my-selector {
  background: $bgColor-default;
}
```

Reply:
Its passing a CSS var as a fallback, so it does respect theme. In this case it will look like `var(--bgColor-default, var(--color-bg-default)`